src/lib/djerba/extract/r_script_wrapper.py

```diff
@@ -273,20 +284,31 @@ def preprocess_fus(self, mavis_path):
             msg = "Found more than one Mavis summary .tab file in "+mavis_path
             raise RuntimeError(msg)
         fus_path = zf.extract(matched[0], self.tmp_dir)
+        
         # prepend column to the extracted summary path
         out_path = os.path.join(self.tmp_dir, 'fus.txt')
+        
+        # Check if the .tab file is empty
+        self.tab_file_is_empty(fus_path) 
+
         with open(fus_path, 'rt') as fus_file, open(out_path, 'wt') as out_file:
-            reader = csv.reader(fus_file, delimiter="\t")
-            writer = csv.writer(out_file, delimiter="\t")
-            in_header = True
-            for row in reader:
-                if in_header:
-                    value = 'Sample'
-                    in_header = False
-                else:
-                    value = self.tumour_id
-                new_row = [value] + row
-                writer.writerow(new_row)
+            if self.is_tab_empty == False:
+                fus_file.seek(0) # go back to the top of the file...I don't think this is necessary, but just in case.
```

Comment:
You are right -- this isn't necessary and can be removed. :grinning: 
The Python `open` method implicitly creates a new `file` object, which is not affected by the status of any other object which might have read the file in the past. You only have to worry about using `seek` if you want to jump around to different locations in a file using the same `file` object (and if multiple processes might be _writing_ to a file at the same time it gets complicated, but that's not a concern here).